monitor-openqa_job

```diff
@@ -31,10 +31,7 @@ declare -A failed_versions
 failed_jobs=()
 for job_id in $(job_ids job_post_response); do
     log-info "Waiting for job $job_id to finish"
-    while sleep "${sleep_time}"; do
-        job_state=$($openqa_cli api --host "$host" jobs/"$job_id" | jq -r '.job.state')
-        [[ $job_state = 'done' ]] && break
-    done
+    $openqa_cli monitor --follow --poll-interval "$sleep_time" "$job_id"
```

Comment:
That doesn't work.
http://jenkins.qa.suse.de/job/monitor-openQA_in_openQA-TW/31952/console
```
+ openqa-cli monitor --follow --poll-interval 10 5138666
404 Not Found
```
It misses the hostname